mtp_cashbook/apps/cashbook/tests/test_views.py

```diff
@@ -232,6 +233,127 @@ def test_new_credits_submit(self, mock_create_transaction):
                 '£52.00' in mail.outbox[1].body and '£45.00' in mail.outbox[0].body
             )
 
+    @override_settings(
+        ENVIRONMENT='prod',  # because non-prod environments don't send to .local
+        PRISONER_CAPPING_ENABLED=True,
+        PRISONER_CAPPING_THRESHOLD_IN_POUNDS=100,
+    )
+    @mock.patch('cashbook.tasks.logger')
+    @mock.patch('cashbook.tasks.nomis')
```

Comment:
Seems kinda weird having `RequestsMock` mock everything except the nomis requests. Any reason for this?

Reply:
it's the same as the other tests in this file. also, this test doesn't return a pre-set value for NOMIS requests, it actually handles the data passed in.

Reply:
Yeah it's true that it's similar to the rest of the file, so I won't block on it